integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -850,6 +850,90 @@ private void createPV() throws Exception {
     new PersistentVolume("/scratch/acceptance_test_pv/persistentVolume-" + domainUid, pvMap);
   }
 
+  public void testDomainServerRestart(String oldPropertyString, String newPropertyString)
+      throws Exception {
+    logger.info("Inside testDomainServerPodRestart");
+    boolean result = false;
+    result =
+        TestUtils.findFileContainString(
+            BaseTest.getUserProjectsDir() + "/weblogic-domains/" + domainUid + "/domain.yaml",
+            newPropertyString);
+    logger.info("The search result for " + newPropertyString + " is: " + result); // TODO
+    if (!result) {
+      TestUtils.createNewYamlFile(
+          BaseTest.getUserProjectsDir() + "/weblogic-domains/" + domainUid + "/domain.yaml",
+          BaseTest.getUserProjectsDir() + "/weblogic-domains/" + domainUid + "/domain_new.yaml",
+          oldPropertyString,
+          newPropertyString);
+      logger.info(
+          "Done - generate new domain.yaml for "
+              + domainUid
+              + " oldProperty: "
+              + oldPropertyString
+              + " newProperty: "
+              + newPropertyString);
+
+      // kubectl apply the new generated domain yaml file with changed property
+      StringBuffer command = new StringBuffer();
+      command
+          .append("kubectl apply  -f ")
+          .append(
+              BaseTest.getUserProjectsDir()
+                  + "/weblogic-domains/"
+                  + domainUid
+                  + "/domain_new.yaml");
+      logger.info("kubectl execut with command: " + command.toString());
+      ExecResult exeResult = ExecCommand.exec(command.toString());
+      if (exeResult.exitValue() != 0) {
+        throw new RuntimeException(
+            "FAILED: command to get domain " + command + " failed with " + exeResult.stderr());
+      }
+      if (!exeResult.stdout().contains(domainUid))
+        throw new RuntimeException("FAILURE: domain not found, exiting!");
+
+      // verify the servers in the domain are being restarted in a sequence
+      verifyAdminServerRestarted();
+      verifyManagedServerRestarted();
+    }
+    logger.info("Done - testDomainServerPodRestart");
+  }
+
+  public void verifyAdminServerRestarted() throws Exception {
+    logger.info("Checking if admin pod(" + domainUid + "-" + adminServerName + ") is Terminating");
+    TestUtils.checkPodTerminating(domainUid + "-" + adminServerName, domainNS);
+    Thread.sleep(10 * 1000);
+    logger.info("Checking if admin pod(" + domainUid + "-" + adminServerName + ") is Running");
+    TestUtils.checkPodCreated(domainUid + "-" + adminServerName, domainNS);
+    Thread.sleep(10 * 1000);
+  }
+
+  public void verifyManagedServerRestarted() throws Exception {
```

Comment:
change method name to verifyManagedServersRestarted as its making sure all managed servers are restarted